data/json/monstergroups.json

```diff
@@ -1052,27 +1047,27 @@
       { "monster" : "mon_zombie_fireman", "freq" : 20, "cost_multiplier" : 2 },
       { "monster" : "mon_dog_zombie_cop", "freq" : 40, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_dog", "freq" : 10, "cost_multiplier" : 2 },
-      { "monster" : "mon_dog_skeleton", "freq" : 5, "cost_multiplier" : 2 },
+      { "monster" : "mon_dog_skeleton", "freq" : 5, "cost_multiplier" : 2, "starts" :   60 },
       { "monster" : "mon_dog_zombie_rot", "freq" : 5, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_grabber", "freq" : 70, "cost_multiplier" : 1 },
-      { "monster" : "mon_skeleton", "freq" : 30, "cost_multiplier" : 2 },
+      { "monster" : "mon_skeleton", "freq" : 30, "cost_multiplier" : 2, "starts" :   60},
       { "monster" : "mon_zombie_smoker", "freq" : 30, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_shady", "freq" : 30, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_gasbag", "freq" : 20, "cost_multiplier": 5},
       { "monster" : "mon_zombie_gasbag", "freq" : 10, "cost_multiplier": 10, "pack_size": [3,5]},
       { "monster" : "mon_zombie_swimmer", "freq" : 20, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_shrieker", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5, "starts" :   120 },
       { "monster" : "mon_zombie_acidic", "freq" : 30, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_electric", "freq" : 30, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_necro", "freq" : 1, "cost_multiplier" : 25 },
       { "monster" : "mon_zombie_survivor", "freq" : 1, "cost_multiplier" : 25 },
       { "monster" : "mon_boomer", "freq" : 30, "cost_multiplier" : 7 },
       { "monster" : "mon_boomer_huge", "freq" : 20, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_brute", "freq" : 10, "cost_multiplier" : 15 },
-      { "monster" : "mon_zombie_hulk", "freq" : 1, "cost_multiplier" : 50 },
-      { "monster" : "mon_skeleton_hulk", "freq" : 1, "cost_multiplier" : 50 },
-      { "monster" : "mon_zombie_master", "freq" : 1, "cost_multiplier" : 25 },
+      { "monster" : "mon_zombie_hulk", "freq" : 1, "cost_multiplier" : 50, "starts" :   180 },
+      { "monster" : "mon_skeleton_hulk", "freq" : 1, "cost_multiplier" : 50, "starts" :   240 },
+      { "monster" : "mon_zombie_master", "freq" : 1, "cost_multiplier" : 25, "starts" :   120 },
       { "monster" : "mon_zombie_brute_shocker", "freq" : 1, "cost_multiplier" : 45 }
```

Comment:
Shocker brutes probably shouldn't be a day 1 spawn

Reply:
Often you can just sneak in the back and break the wall down to the vault so I don't really see the point unless they spawn right inside next to the vault